libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSDKCryptoUtils.h

```diff
@@ -79,6 +79,44 @@ extern NSUInteger const kSFPBKDFDefaultSaltByteLength;
                        derivationRounds:(NSUInteger)numDerivationRounds
                               keyLength:(NSUInteger)derivedKeyLength;
 
+/**
+ * Encrypt the given data using the AES algorithm.
+ * @param data The data to encrypt.
+ * @param key The encryption key used to encrypt the data.
+ * @param keyLength The encryption key length used for key.
+ * @param iv The initialization vector data used for the encryption.
+ * @return The encrypted data, or `nil` if encryption was not successful.
+ */
++ (nullable NSData *)aesEncryptData:(NSData *)data withKey:(NSData *)key keyLength:(NSInteger)keyLength iv:(NSData *)iv;
```

Comment:
@liguopku Could we expose `aes256EncryptData` and `aes256DecryptData` instead of exposing these 2 in public API?

Reply:
yea sure. could we also expose `aes128EncryptData` and `aes128EncryptData`?

Reply:
Sure.

Reply:
👍